project.clj

```diff
@@ -23,6 +23,13 @@
   :url "https://github.com/zero-one-group/geni"
   :license {:name "Apache License"
             :url  "https://www.apache.org/licenses/LICENSE-2.0"}
+  :plugins [[lein-cljfmt "0.7.0"]]
```

Comment:
I believe we should put this and the `:cljfmt` key on the `:dev` profile, as we would not need it in the uberjar for the releases.

Reply:
You are right.
Fixed.